lib/inertia_rails/controller.rb

```diff
@@ -4,6 +4,13 @@ module InertiaRails
   module Controller
     extend ActiveSupport::Concern
 
+    included do
+      before_action do
+        # :inertia_errors are deleted from the session by the middleware
+        InertiaRails.share(errors: session[:inertia_errors]) if session[:inertia_errors].present?
```

Comment:
here we automatically share the `session[:inertia_errors]` into an `errors` prop